h2o-py/tests/pyunit_utils/utilsPY.py

```diff
@@ -3643,6 +3643,20 @@ def build_save_model_GLM(params, x, train, respName):
     model.download_mojo(path=TMPDIR)    # save mojo
     return model
 
+def build_save_model_GAM(params, x, train, respName):
```

Comment:
I'd personally create a common file in the pyunit folder with GAM tests and not put it into test common utils, but it's not really a big issue.

Reply:
I actually write a build_save_model_generic that can be shared among different algos